server/tests/message.integration.spec.js

```diff
@@ -28,6 +28,7 @@ const testMessageObject = {
 
 const testMessageArray = [];
 const fromArray = ['user0','user1','user2','user3'];
+const MessageUnscoped = Message.unscoped();

```

Comment:
@ElijahGeorge what happens if the DB changes between when `MessageUnscoped` is first created? This is the result of a function call, not an alias for a function call.

Reply:
@jsachs, the approaches I have tried are simply what I have learned from studying the sequelize documentation here http://docs.sequelizejs.com/manual/tutorial/scopes.html. I am happy to get on a call with you to get further clarification on what you think might be a better approach